accounts/views.py

```diff
@@ -1069,3 +1066,35 @@ def pending(request):
     }
     tvars.update(paginate(request, pendings, settings.SOUNDS_PENDING_MODERATION_PER_PAGE))
     return render(request, 'accounts/pending.html', tvars)
+
+
+@csrf_exempt
+def donation_complete(request):
+    params = {'cmd': '_notify-validate'}
+    for key, value in request.POST.items():
+        params[key] =  value
+    if settings.DEBUG:
+        paypal_validation_url = "https://www.sandbox.paypal.com/cgi-bin/webscr"
+    else:
+        paypal_validation_url = "https://www.paypal.com/cgi-bin/webscr"
+    req = requests.post(paypal_validation_url, data=params)
+    if req.text == 'VERIFIED':
+        campaign = DonationCampaign.objects.order_by('date_start').last()
```

Comment:
I think it is safe to assume that there will be always one campaign. However I'm not sure I understand @bdejong comment. What does paypal know about our campaigns? As far as I understans campaigns are something we use to simply "group" the donations.